src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+/* region card */
+
+.catalog {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--color-white);
+  
+  &__line1 {
```

Comment:
don't add line wrappers - just set `width` and `flex-wrap` instead
P.S. don't forget to remove `flex-direction`